### ISSUE TITLE
ssl_verify does not accept a cert file (self-signed)

### ISSUE THREAD

**Issue by aram535** [vavarachen/splunk_http_handler#16]
Hi, seems like the ssl_verify parameter is just True or False.  With False, there doesn't seem to be a way to suppress the "warnings".  

Normally you can provide True, or a cert to be used as the chain of the certification. 

The other option is add a requests.session as a parameter so that the user can provide their own session with the certificate already in the chain.

**Comment by vavarachen** [comment]
Yeah, this aspect is not well documented.  You can specify the path to the
cert instead of True/False.

https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification:~:text=You%20can%20pass,%3E%3E%3E
